### DISCUSSION TITLE
GitHub Account Suspension - Seeking Help

### DISCUSSION THREAD

**Discussion by gmdcodes** [community/community#120521] [Discussions]
### Select Topic Area

Question

### Body

My GitHub account, rising-gmd, was suspended without warning despite no violations. I submitted a ticket (Ticket ID: 2734250) but need help. When can I expect a response?

**Comment by LiteBrite82** [comment]
Hi @gmdcodes  

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
